sn_node/src/log_markers.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use libp2p::{kad::RecordKey, PeerId};
+use sn_protocol::{
+    messages::PaymentProof,
+    storage::{ChunkAddress, SpendWithParent},
+};
+use std::time::Duration;
+// this gets us to_string easily enough
+use strum::Display;
+
+/// Public Markers for generating log output,
+/// These generate apprioriate log level output and consistent strings.
+/// Changing these log markers is a breaking change.
+#[derive(Debug, Clone, Display)]
+pub enum Marker<'a> {
+    /// The node has started
+    NodeConnectedToNetwork,
+
+    /// No network activity in some time
+    NoNetworkActivity(Duration),
+
+    /// Put request received from + for
+    KademliaPutRequest {
+        /// The peer that sent the request
+        peer: PeerId,
+        /// The key of the record being put
+        key: &'a RecordKey,
+    },
```

Comment:
This will not be triggered with our current setup, as we don't have kad's PUT/replication enabled. 
Maybe we can add this later when it is required?

Reply:
updated for cmd in/out